docs/source/metadata.rst

```diff
@@ -49,18 +91,56 @@ returned. If the ``in_place`` flag is ``False``, only the altered coordinates ar
 
 	coordinates = ((10, 10), (10, 100), (200, 100), (200, 10))
 	coordinate_system = PixelSpace(width=850, height=1100)
+
 	element = Element(coordinates=coordinates, coordinate_system=coordinate_system)
 	print(element.metadata.coordinates.to_dict())
 	print(element.metadata.coordinates.system.orientation)
 	print(element.metadata.coordinates.system.width)
 	print(element.metadata.coordinates.system.height)
+
 	element.convert_coordinates_to_new_system(RelativeCoordinateSystem(), in_place=True)
 	# Should now be in terms of new coordinate system
 	print(element.metadata.coordinates.to_dict())
 	print(element.metadata.coordinates.system.orientation)
 	print(element.metadata.coordinates.system.width)
 	print(element.metadata.coordinates.system.height)
 
+###########################################
+Additional Metadata Fields by Document Type
+###########################################
+
++-------------------------+---------------------+--------------------------------------------------------+
+| ``Field Name``          | Applicable Doc Types| Short Description                                      |
```

Comment:
Field Name should not be monospace, because it isn't referring to code:
```suggestion
| Field Name          | Applicable Doc Types| Short Description            |
```